circle/docker-image-test.sh

```diff
@@ -28,19 +28,32 @@ if [[ -d .circleci/scripts/pre-tests.d/ ]]; then
   done
 fi
 
-if [[ -n $RELEASE_SERIES_LIST ]]; then
+if [[ -z $RELEASE_SERIES_LIST ]]; then
+  docker_build $DOCKER_PROJECT/$IMAGE_NAME . || exit 1
+else
+  IFS=',' read -ra DISTRIBUTIONS_ARRAY <<< "${DISTRIBUTIONS_LIST:-debian-8}"
   IFS=',' read -ra RELEASE_SERIES_ARRAY <<< "$RELEASE_SERIES_LIST"
-  for RS in "${RELEASE_SERIES_ARRAY[@]}"; do
-    if [[ -n $IMAGE_TAG ]]; then
-      if [[ "$IMAGE_TAG" == "$RS"* ]]; then
-        docker_build $DOCKER_PROJECT/$IMAGE_NAME:$RS $RS || exit 1
+  for distro in "${DISTRIBUTIONS_ARRAY[@]}"; do
+    for RS in "${RELEASE_SERIES_ARRAY[@]}"; do
```

Comment:
just a suggestion, you could be consistent in the naming and user `rs` instead of `RS`